pkg/driver/controller.go

```diff
@@ -86,14 +86,17 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		}
 	}
 
+	volumeParams := req.GetParameters()
+	fsType := volumeParams["fsType"]
+
 	// volume exists already
 	if disk != nil {
+		disk.FsType = fsType
```

Comment:
I'm wondering if it's necessary for us to return the FS type as an attribute. The reason is that we don't do any validation with it, we simple send it back to the CO.

For example, if the client sends in an invalid FS type, we return success. Later on it'll fail when staging, but I'm wondering if we should return an error right here. Another option would be to ignore it (other drivers seems to do that).

@leakingtapan, do you know if it's strictly necessary that we return this info back to the CO?

Reply:
We need to pass `fsType` back since it is needed by `NodeStageVolume` later [here](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/pkg/driver/node.go#L100). 


> Another option would be to ignore it (other drivers seems to do that).

In other driver, what's the behavior once a invalid fsType is ignored? Does it fall back to default option?

Reply:
> We need to pass fsType back since it is needed by NodeStageVolume later here.

From what I understand from CreateVolume specs, that wouldn't be required. 

I took a quick look in GCP and it seems like they don't handle fsType as well. It'd be nice to test this.

It'd be nice to have this tested, but we can probably do that in another PR.

Reply:
> From what I understand from CreateVolume specs, that wouldn't be required.
It's not required in `CreateVolume` but then when the volume is formatted in `NodeStageVolume` the fsType is missing and it would just use the default right?

Reply:
@bertinatto it is not required to be CSI compliant, however, it is required to be EBS in-tree complicant since `fsType` is supported there. See [here](https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs). And this explains why it is not handled by GCE PD CSI driver since their in-tree doesn't have this. See [here](https://kubernetes.io/docs/concepts/storage/storage-classes/#gce-pd)

@dkoshkin Yep. When `fsType` is missing the default one will be used in node stage phase [here](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/pkg/driver/node.go#L102)

So from a EBS point of view, we would still need `fsType` passed into `NodeStageVolume` from `CreateVolume`